scripting/get5/natives.sp

```diff
@@ -139,15 +139,34 @@ public int Native_AddPlayerToTeam(Handle plugin, int numParams) {
 public int Native_SetPlayerName(Handle plugin, int numParams) {
   char auth[AUTH_LENGTH];
   char name[MAX_NAME_LENGTH];
+  KeyValues namesKv = new KeyValues("Names");
+  char nameFile[] = "get5_names.txt";
```

Comment:
I think this approach has some problems.

1. In terms of code health, there is duplicate code with this function and `LoadPlayerNames` now. I don't see a clear reason why there needs to be two places that implement this code.

2. If multiple server processes are running on the server installation the `get5_names.txt` files will clash and you'll get a weird file.

Barring some unknown valve server bug, I don't see any clear reason to try to be efficient with the file building/loading.

Reply:
Sounds good! Just wasn't sure which approach would be best (reloading everyone's name vs. appending and updating). I'll revert the last few commits to just call `LoadPlayerNames()` inside the `SetPlayerName` call then :)